ckanext/opendata_theme/opengov_custom_homepage/controller.py

```diff
@@ -0,0 +1,86 @@
+# encoding: utf-8
+import ast
+from collections import OrderedDict
+
+import ckan.controllers.admin as admin
+import ckan.lib.navl.dictization_functions as dict_fns
+from ckan.logic import (
+    clean_dict,
+    tuplize_dict,
+    parse_params
+)
+from ckan.plugins.toolkit import (
+    get_action,
+    redirect_to,
+    render,
+    request
+)
+
+from ckanext.opendata_theme.opengov_custom_homepage.constants import LAYOUTS
+from ckanext.opendata_theme.opengov_custom_homepage.processor import custom_naming_processor
+
+
+class CustomCSSController(admin.AdminController):
```

Comment:
The class in the custom homepage controller should be renamed to `CustomHomepageController`.

Reply:
checked and tested it.
it is impossible to store anything except dict in the config, and get from backend anything than string which is not a valid json because it is stored as a somehow pickled object. so can not change it.